### PR TITLE
Community: Node-RED Messages node

### PR DESCRIPTION
**Pull request by danpeig**
This is another community project that implements Meshtastic.js in the form of a Node-RED node allowing users to send and receive messages / packets using a device connected via MQTT.

### CODE REVIEW

File: docs/community/software/node-red-messages.mdx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+- Send packets to any Meshtastic APP (port num)
+- Plug and Play: no additional servers, no containers, no CLI, no binary files, super simple configuration
+- Indirect support to MQTT via uplink/downlink channels
+
+## Limitations
+- Connect directly to the MQTT server: this is not supported by [Meshtastic.js](https://js.meshtastic.org/)
+- Connect to a device via Bluetooth or Serial (not implemented yet)
+
+## Instructions
+For installation and usage instructions, please visit the project page at [GitHut](https://github.com/danpeig/node-red-meshtastic-msg).
```
Comment (pdxlocations) [inline docs/community/software/node-red-messages.mdx:34]: typo 'GitHut"

**Review by pdxlocations — COMMENTED** [review]
Looks like an exciting project!

File: docs/community/software/node-red-messages.mdx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+## Screenshot
+![Node-RED Meshtastic Messages node](https://github.com/danpeig/node-red-meshtastic-msg/raw/main/resources/flow_example.png)
```
Comment (pdxlocations) [inline docs/community/software/node-red-messages.mdx:37]: I believe it's preferred to place images in
https://github.com/meshtastic/meshtastic/tree/master/static/img

Also webp format is preferred\.

File: docs/community/software/node-red-messages.mdx (hunk 1 above)
Comment (danpeig) [reply docs/community/software/node-red-messages.mdx]: Fixed!

File: docs/community/software/node-red-messages.mdx (hunk 2 above)
Comment (danpeig) [reply docs/community/software/node-red-messages.mdx]: Fixed!

**Review by pdxlocations — APPROVED** [review]

**Comment by pdxlocations** [comment]
Oh whoops, looks like your PR included a [package-lock.json](https://github.com/meshtastic/meshtastic/pull/1374/files#diff-053150b640a7ce75eff69d1a22cae7f0f94ad64ce9a855db544dda0929316519) file and I merged it along with your new content.

**Comment by pdxlocations** [comment]
@danpeig would you mind submitting another PR to remove this extra file?

**Comment by danpeig** [comment]
Oh...gitignore should not let this happen. Will fix that.

**Pull request merged**